archinstall/lib/interactions/disk_conf.py

```diff
@@ -407,8 +428,12 @@ def suggest_multi_disk_layout(
 	boot_partition = _boot_partition(root_device_sector_size, using_gpt)
 	root_device_modification.add_partition(boot_partition)
 
+	# add root partition to the root device
 	root_start = boot_partition.start + boot_partition.length
-	root_length = root_device.device_info.total_size - root_start
+	available_space = root_device.device_info.total_size - root_start
+	root_length = process_root_partition_size(
```

Comment:
I'm a bit confused by this logic, above we're saying that the minimum root size needs to be 20GB and then we look for a device that comes closest to that number. Now here we'll determine the actual root size. 
So lets say there's devices with 25GB, 30GB, 50GB. Then it'll pick the 25GB one, and based on the root size logic it'll get set to 20GB. Wouldn't it make more sense to actually utilize the entire disk?

Reply:
Hmm yes I see your point, for a 25GB it feels a bit ridiculous to leave 5GB empty.

My idea was that for a 200GB disk, we'll take 20GB and leave the rest unused, so that you can resize it however you want later on.

In the end it's a matter of choice, what do you think should be the best strategy? Do you have a high level algorithm in mind?

Reply:
Maybe we just leave the multi disk suggestion as is for now and update only the single one, I think that one is the one causing the most headaches for people at the moment